firmware/examples/TaskScheduler/source/main.cpp

```diff
@@ -39,15 +39,27 @@ class PrinterTask : public rtos::Task<1024>
 
 int main()
 {
+    const char * task_names[10] = {
+        "Printer A",
+        "Printer B"
+    };
+    const char * messages[20] = {
+        "I am a printer, I am faster",
+        "I am also a printer"
+    };
+    char printer_c_name[]     = "Printer C";
+    char message[]            = "I do not exist";
+
     DEBUG_PRINT("Starting TaskScheduler example...");
-    // TODO(#210): Should remove the use of new and use static allocation for
-    //             constructing tasks when the issue is fixed.
-    PrinterTask * printer_one =
-        new PrinterTask("Printer A", "I am a printer, I am faster");
-    printer_one->SetDelayTime(500);
-    PrinterTask * printer_two =
-        new PrinterTask("Printer B", "I am also a printer");
-    printer_two->SetDelayTime(1000);
+
+    PrinterTask printer_one(task_names[0], messages[0]);

```

Comment:
The developer should be able to put a string literal here, without needing to declare it as a C-style array above.